test/unit/fixture/dedupe_elasticsearch_results.js

```diff
@@ -1,377 +1,1192 @@
 module.exports = [
   {
     'center_point': {
-      'lon': -76.207456,
-      'lat': 40.039265
+      'lon': -76.293127,
+      'lat': 40.032787
     },
-    'address_parts': {},
     'parent': {
-      'localadmin': ['East Lampeter'],
-      'region_a': ['PA'],
-      'region': ['Pennsylvania'],
-      'locality': ['Smoketown'],
-      'country_a': ['USA'],
-      'county': ['Lancaster County'],
-      'country': ['United States'],
-      'neighbourhood': ['Greenland']
+      'country': [
+        'United States'
+      ],
+      'macrocounty_a': [],
+      'locality_a': [
+        null
+      ],
+      'county': [
+        'Lancaster County'
+      ],
+      'borough_a': [],
```

Comment:
I thought we got rid of all these empty arrays coming back from ES?

If you're using `pelias/model` to generate them you can use `doc.toESDocument()` to strip empty arrays from the output.